packages/dd-trace/src/format.js

```diff
@@ -143,6 +142,9 @@ function addTag (meta, metrics, key, value, seen) {
       if (isNaN(value)) break
       metrics[key] = value
       break
+    case 'boolean':
+      metrics[key] = value ? 1 : 0
```

Comment:
Can you add a quick test to add the missing coverage here?

Reply:
Done.